### PR TITLE
Initiating release of 0.15.0

### PR DESCRIPTION
**Pull request by dfarrell07**

### CODE REVIEW

File: releases/v0.15.0.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+version: v0.15.0
+name: 0.15.0
+branch: release-0.15
+status: shipyard
+components:
+  shipyard: 3236ac72011806d4a855dc7847c67810b2c13f33
```
Comment (dfarrell07) [inline releases/v0.15.0.yaml]: :heavy_check_mark: https://github.com/submariner-io/shipyard/commit/3236ac72011806d4a855dc7847c67810b2c13f33

**Comment by dfarrell07** [comment]
> ERROR: Failed to determine latest commit hash for shipyard

https://github.com/submariner-io/releases/actions/runs/4851546142/jobs/8645493936?pr=706

Re-running

**Comment by dfarrell07** [comment]
Just realized auto-merging Dependabot are a potential issue, wondering if we should let that smoke clear.

File: releases/v0.15.0.yaml (hunk 1 above)
Comment (skitt) [inline releases/v0.15.0.yaml]: This is now

```suggestion
  shipyard: 4b02307e4332a05dbfe99b3a6707314c01680424
```

**Review by sridhargaddam — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**